### ISSUE TITLE
Using Fade, Wrap Around and Auto Play

### ISSUE THREAD

**Issue by aaronstezycki** [metafizzy/flickity#1007]
Using 'Fade', 'Wrap Around' and 'Auto Play' together when only 2 items exist in the slider causes a visual bug when transitioning between slides. Test case provided below. The issue rectifies itself when more than two slides exist.

It's a very specific use case, but never the less is still a problem that should be fixed. I don't have the means to look into this myself, so hoping the greater community can pose a suggestion/fix.

**Test case:** https://codepen.io/aaron-stez/pen/NWPxapR

**Comment by AllAstronauts** [comment]
Bump. Can confirm with Fade option.

**Comment by aaronstezycki** [comment]
@AllAstronauts Yeah unfortunately for us, the slider is used to deliver images with content in a banner. Generally promotions or sales type content, and on the odd occasion there is just two slides. So I guess it all depends on who your viewership are and what they are consuming.
